README.md

```diff
@@ -232,12 +240,12 @@ The command-line tool can be used to anonymize a folder containing FHIR resource
 
 Example usage to anonymize FHIR resource files in a folder: 
 ```
-> .\Fhir.Anonymizer.Tool.exe -i myInputFolder -o myOutputFolder
+> .\Fhir.Anonymizer.<version>.Tool.exe -i myInputFolder -o myOutputFolder
 ```
 
 ## Configuration file format
 
-The configuration is specified in JSON format. It has two high-level sections. One of these sections, namely _fhirPathRules_ is meant to specify de-identification methods for data elements. The second section named _parameters_ affects global behavior. _fhirPathRules_ are executed in the order of appearance in the configuration file.
+The configuration is specified in JSON format. It has two high-level sections. One of these sections, namely _fhirPathRules_ is meant to specify de-identification methods for data elements. The second section named _parameters_ affects global behavior. _fhirPathRules_ are executed in the order of appearance in the configuration file. The content of _fhirPathRules_ is version dependent since resources are different in R4 and Stu3. 

```

Comment:
We can remove this explanation as FHIRPath itsself is independent from R4 and STU 3?